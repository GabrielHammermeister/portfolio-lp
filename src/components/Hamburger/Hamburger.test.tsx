/* eslint-disable @typescript-eslint/no-empty-function */
import { render } from '@testing-library/react'
import Hamburger from './Hamburger.index'

describe('<Hamburger />', () => {
   it('should render the heading', () => {
      const layout = render(<Hamburger handleOnClick={() => {}} />)

      expect(layout).toMatchSnapshot()
   })
})
