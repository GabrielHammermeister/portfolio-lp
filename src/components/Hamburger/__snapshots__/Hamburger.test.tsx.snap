// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hamburger /> should render the heading 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c0 {
  position: absolute;
  right: 1rem;
  top: 1rem;
  z-index: 11;
  outline: none;
  border: none;
  cursor: pointer;
  width: 50px;
  height: 50px;
  border-radius: 50%;
  background: transparent;
}

.c2 {
  text-align: left;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  list-style: none;
  gap: 30px;
  margin-left: 50px;
}

.c2 a {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c3 {
  color: inherit;
}

.c1 {
  background-color: #FFFFFF;
  position: absolute;
  z-index: 10;
  right: 0.5rem;
  top: 0.5rem;
  height: 100px;
  width: 100px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media (min-width:768px) {
  .c0 {
    right: 5rem;
    top: 2rem;
  }

  .c0 svg {
    width: 2rem;
    height: 2rem;
  }
}

@media (min-width:768px) {
  .c1 {
    right: 4rem;
    top: 1rem;
  }
}

<body>
    <div>
      <button
        class="c0"
      >
        <svg
          height="23"
          viewBox="0 0 23 23"
          width="23"
        >
          <path
            fill="transparent"
            stroke="white"
            stroke-linecap="square"
            stroke-width="1.5"
          />
          <path
            d="M 2 9.423 L 20 9.423"
            fill="transparent"
            stroke="white"
            stroke-linecap="square"
            stroke-width="1.5"
          />
          <path
            fill="transparent"
            stroke="white"
            stroke-linecap="square"
            stroke-width="1.5"
          />
        </svg>
      </button>
      <div
        class="c1"
      >
        <ul
          class="c2"
        >
          <a
            href="#About"
          >
            <li
              class="c3"
            >
              About me
            </li>
          </a>
          <a
            href="#Portifolio"
          >
            <li
              class="c3"
            >
              Portifolio
            </li>
          </a>
          <a
            href="#Email"
          >
            <li
              class="c3"
            >
              Mail me
            </li>
          </a>
        </ul>
      </div>
    </div>
  </body>,
  "container": .c0 {
  position: absolute;
  right: 1rem;
  top: 1rem;
  z-index: 11;
  outline: none;
  border: none;
  cursor: pointer;
  width: 50px;
  height: 50px;
  border-radius: 50%;
  background: transparent;
}

.c2 {
  text-align: left;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  list-style: none;
  gap: 30px;
  margin-left: 50px;
}

.c2 a {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c3 {
  color: inherit;
}

.c1 {
  background-color: #FFFFFF;
  position: absolute;
  z-index: 10;
  right: 0.5rem;
  top: 0.5rem;
  height: 100px;
  width: 100px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media (min-width:768px) {
  .c0 {
    right: 5rem;
    top: 2rem;
  }

  .c0 svg {
    width: 2rem;
    height: 2rem;
  }
}

@media (min-width:768px) {
  .c1 {
    right: 4rem;
    top: 1rem;
  }
}

<div>
    <button
      class="c0"
    >
      <svg
        height="23"
        viewBox="0 0 23 23"
        width="23"
      >
        <path
          fill="transparent"
          stroke="white"
          stroke-linecap="square"
          stroke-width="1.5"
        />
        <path
          d="M 2 9.423 L 20 9.423"
          fill="transparent"
          stroke="white"
          stroke-linecap="square"
          stroke-width="1.5"
        />
        <path
          fill="transparent"
          stroke="white"
          stroke-linecap="square"
          stroke-width="1.5"
        />
      </svg>
    </button>
    <div
      class="c1"
    >
      <ul
        class="c2"
      >
        <a
          href="#About"
        >
          <li
            class="c3"
          >
            About me
          </li>
        </a>
        <a
          href="#Portifolio"
        >
          <li
            class="c3"
          >
            Portifolio
          </li>
        </a>
        <a
          href="#Email"
        >
          <li
            class="c3"
          >
            Mail me
          </li>
        </a>
      </ul>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`
